tests/python/frontend/paddlepaddle/test_forward.py

```diff
@@ -1681,5 +1687,15 @@ def forward(self, inputs, prev_h):
         )
 
 
+@tvm.testing.uses_gpu
+def test_forward_topk():
+    @paddle.jit.to_static
+    def topk1(inputs):
+        return paddle.topk(inputs, k=1)
+
+    input_data = paddle.to_tensor([1, 4, 5, 7])
+    verify_model(topk1, input_data=input_data)
```

Comment:
You can add a few different cases, example:
1. k is a tensor and not a int
2. set largest/axis/sort attribute 
3. The output is only the values or the indices.

Reply:
according to paddle.topk api, there seems is no a argument to control the output that only have the values or indices

Reply:
Yes, the paddle.topk api will definitely have two outputs.